doc/trajopt_planner/trajopt_planner_tutorial.rst

```diff
@@ -0,0 +1,123 @@
+TrajOpt Planner
```

Comment:
In trajopt.png, the function was renamed from hatch, should change it there as well.

Reply:
right, thanks